packages/caliper-core/lib/manager/monitors/monitor-prometheus.js

```diff
@@ -110,43 +116,41 @@ class PrometheusMonitor extends MonitorInterface {
     async getStatistics(testLabel) {
         this.endTime = Date.now()/1000;
 
-        if (this.include) {
+        if (this.queries) {
             const resourceStats = [];
             const chartArray = [];
-            for (const metricKey of Object.keys(this.include)) {
-                let newKey = true;
+            for (const queryObject of this.queries) {
                 // Each metric is of the form
-                // Tag0: {
+                // {
+                //     name: 'tag name',
                 //     query: 'the prometheus query to be made',
                 //     statistic: 'the action to be taken on returned metrics'
                 //     step: step size
                 // }
-                const params = this.include[metricKey];
-                const queryString = PrometheusQueryHelper.buildStringRangeQuery(params.query, this.startTime, this.endTime, params.step);
+                const queryString = PrometheusQueryHelper.buildStringRangeQuery(queryObject.query, this.startTime, this.endTime, queryObject.step);
                 const response = await this.prometheusQueryClient.getByEncodedUrl(queryString);
 
                 // Retrieve base mapped statistics and coerce into correct format
-                const resultMap = PrometheusQueryHelper.extractStatisticFromRange(response, params.statistic, params.label);
+                const resultMap = PrometheusQueryHelper.extractStatisticFromRange(response, queryObject.statistic, queryObject.label);
 
                 const metricArray = [];
+                let newQuery = true;
                 for (const [key, value] of resultMap.entries()) {
```

Comment:
can be be a bit more descriptive on what a key really is and what the value might be ?